feature.env

```diff
@@ -12,6 +12,7 @@ FRONTEND_COMPONENT_API_URL=http://localhost:9091
 
 STUB_PRISONER_LIST_PAGE_ENABLED=true
 FRONTEND_COMPONENTS_API_FEATURE_TOGGLE_ENABLED=true
+ADD_MULTIPLE_GOALS_ENABLED=true
```

Comment:
The toggle is enabled in our feature env file, so is ready for cypress tests as and when we get there.
All other envs currently have it disabled